src/docs/development/ui/layout/index.md

```diff
@@ -1,1191 +1,52 @@
-      ),
-      Expanded(
-        child: Image.asset('images/pic3.jpg'),
-      ),
-    ],
-  );
-  {% endprettify %}
-</div>
-<div class="col-lg-4" markdown="1">
-  {% asset ui/layout/row-expanded-visual.png class="mw-100"
-      alt="Row of 3 images with the middle image twice as wide as the others" %}
-
-  **App source:** [sizing]({{examples}}/layout/sizing)
-</div>
-</div>
-
-[sizing]: {{examples}}/layout/sizing
-
-### Packing widgets
-
-By default, a row or column occupies as much space along its main axis
-as possible, but if you want to pack the children closely together,
-set its `mainAxisSize` to `MainAxisSize.min`. The following example
-uses this property to pack the star icons together.
-
-<div class="row">
-<div class="col-lg-8">
-  <?code-excerpt "layout/pavlova/lib/main.dart (stars)" replace="/mainAxisSize.*/[!$&!]/g; /\w+ \w+ = //g; /;//g"?>
-  {% prettify dart context="html" %}
-  Row(
-    [!mainAxisSize: MainAxisSize.min,!]
-    children: [
-      Icon(Icons.star, color: Colors.green[500]),
-      Icon(Icons.star, color: Colors.green[500]),
-      Icon(Icons.star, color: Colors.green[500]),
-      Icon(Icons.star, color: Colors.black),
-      Icon(Icons.star, color: Colors.black),
-    ],
-  )
-  {% endprettify %}
-</div>
-<div class="col-lg-4" markdown="1">
-  {% asset ui/layout/packed.png class="border mw-100"
-      alt="Row of 5 stars, packed together in the middle of the row" %}
-
-  **App source:** [pavlova]({{examples}}/layout/pavlova)
-</div>
-</div>
-
-### Nesting rows and columns
-
-The layout framework allows you to nest rows and columns inside of rows
-and columns as deeply as you need. Let's look the code for the outlined section
-of the following layout:
-
-{% asset ui/layout/pavlova-large-annotated.png class="border mw-100"
-    alt="Screenshot of the pavlova app, with the ratings and icon rows outlined in red" %}
-{:.text-center}
-
-The outlined section is implemented as two rows. The ratings row contains
-five stars and the number of reviews. The icons row contains three
-columns of icons and text.
-
-The widget tree for the ratings row:
-
-{% asset ui/layout/widget-tree-pavlova-rating-row.png class="mw-100" alt="Ratings row widget tree" %}
-{:.text-center}
-
-The `ratings` variable creates a row containing a smaller row of 5 star icons,
-and text:
-
-<?code-excerpt "layout/pavlova/lib/main.dart (ratings)" replace="/ratings/[!$&!]/g"?>
-```dart
-var stars = Row(
-  mainAxisSize: MainAxisSize.min,
-  children: [
-    Icon(Icons.star, color: Colors.green[500]),
-    Icon(Icons.star, color: Colors.green[500]),
-    Icon(Icons.star, color: Colors.green[500]),
-    Icon(Icons.star, color: Colors.black),
-    Icon(Icons.star, color: Colors.black),
-  ],
-);
+This document doesn't cover how to develop Flutter apps or explain how to code in Dart. It assumes you have a functioning knowledge of object-oriented programming. However, don't be intimidated to read through this document and interact with the code demos.   
```

Comment:
Break up into lines that are 80 chars or less.

Reply:
1. Changed. 
2. I would never want to replace something you wrote, unless you told me to :)
3. Working on that. Looks like I'll be presenting today. 
4. Noted. 
5. That's right. 
6. I expected this comment. And I was hoping I'd get some feedback on my style. I'll definitely refer to this section on my next pass.